main.go

```diff
@@ -78,6 +78,7 @@ func main() {
 	resolvers.Register("workitem.update", collector.ConfiguredVars(config, collector.NewWorkItemResolver(witClient)))
 	resolvers.Register("comment.create", collector.ConfiguredVars(config, collector.NewCommentResolver(witClient)))
 	resolvers.Register("comment.update", collector.ConfiguredVars(config, collector.NewCommentResolver(witClient)))
+	resolvers.Register("user.email.update", collector.ConfiguredVars(config, collector.NewUserResolver(witClient)))
```

Comment:
Missing template ?

Reply:
added a fork of workitem.update template where I stripped out almost everything. Will make it look pretty  https://github.com/fabric8-services/fabric8-notification/pull/51/commits/8f420859f8488cf74226e5c2171becb511f8b671